torch/distributed/_shard/sharded_tensor/api.py

```diff
@@ -39,7 +39,7 @@
 _sharded_tensor_map: Dict[int, 'weakref.ReferenceType[ShardedTensor]'] = {}
 
 # Custom sharded ops
-_SHARDED_OPS: Dict[str, Callable] = {}
+_SHARDED_OPS: Dict[Callable, Callable] = {}
```

Comment:
should this `_SHRDED_OPS` be removed, and we consolidate the ops into the base `ShardingSpec` instead?

Reply:
See comment above, it doesn't seem right to have ops registered on an interface class.